### ISSUE TITLE
Retenue - Problème des dates

### ISSUE THREAD

**Issue by mprn17** [doc212/ades#112]
Si l'on souhaite mettre en retenue un élève, alors que la retenue est déjà terminée > afficher un pop-up " Attention cette retenue est déjà passée ? " " Souhaitez-vous tout de même mettre cet(te) élève en retenue ? "

**Comment by doc212** [comment]
Et quand on crée une date de retenue dans le passé ? Il faudrait émettre un avertissement aussi non ?

**Comment by doc212** [comment]
Et sur le même sujet, on devrait pas cacher automatiquement les retenues qui sont dans le passé ?

Comment vous utilisez cette fonctionnalité de cacher les retenues ?

**Comment by mprn17** [comment]
Oui, oui c'est une très bonne idée. En fait, jusque maintenant, nous devions retourner dans le menu Retenues>Liste et la décocher (visible).
C'est mieux de cacher automatiquement, celles qui sont passées !
😊
Date: Mon, 8 Jun 2015 17:17:15 -0700

**Comment by mprn17** [comment]
Oui, oui.
Parfois, on "rattrape" des retenues en les encodant a posteriori. Parce qu'on n'a pas eu le temps de les encoder pour diverses raisons en fait.
Mais, il est important qu'on puisse les encoder après coup !


Date: Mon, 8 Jun 2015 15:55:28 -0700
